mimic/test/test_maas.py

```diff
@@ -228,6 +228,26 @@ def test_partial_update_entity(self):
         self.assertEquals('ag13378901234', data['agent_id'])
         self.assertEquals('ItsAnEntity', data['label'])
 
+    def test_get_alarm(self):
+        """
+        test getting an alarm by ID
```

Comment:
Test assertions should be described as a description of the behavior involved, and not as an explanation of what the test is doing.  For example, if you have to use the words "test that", "verify that", or "should", it's a hint you're writing it incorrectly.  https://github.com/testing-cabal/testdoc sort of explains this although I am pretty sure @jml actually wrote something longer about it at some point and I would love it if he magically showed up to link to it...

In any case, here, the appropriate translation would be something like "The URL `/entities/<entity_id>/alarms/<alarm_id>` returns a JSON description of the alarm".